icu4c/source/test/intltest/intltest.cpp

```diff
@@ -241,6 +230,10 @@ UnicodeString toString(UBool b) {
   return b ? UnicodeString("true"):UnicodeString("false");
```

Comment:
would it make sense to print the actual value if it's not 0 or 1, in case someone does assertEquals with tiny ints that are not booleans?